TeamSnapSDK/SDK/Conveniences/NSURL+TSDKConveniences.m

```diff
@@ -0,0 +1,21 @@
+//
+//  NSURL+TSDKConveniences.m
+//  TeamSnapSDK
+//
+//  Created by Jason Rahaim on 11/2/16.
+//  Copyright © 2016 teamsnap. All rights reserved.
+//
+
+#import "NSURL+TSDKConveniences.h"
+
+@implementation NSURL (TSDKConveniences)
+
+- (NSURL *)URLByAppendingQueryParameter:(NSString *)queryString {
```

Comment:
This duplicates established functionality in Apple's Libraries. See https://developer.apple.com/reference/foundation/nsurlqueryitem for building query strings.